### PR TITLE
Remove PropTypes from MaskedViewIOS.ios.js

### PR DESCRIPTION
**Pull request by nd-02110114**
related #21342 

This is a first PR for this repo. 
So, if there are any problem, please tell me 🙇 

TODO
* delete props types 
* apply read only interface

CheckList
 - [x] `yarn prettier`
 - [x] `yarn flow-check-ios`

### CODE REVIEW

File: Libraries/Components/MaskedView/MaskedViewIOS.ios.js [hunk 1]
```diff
@@ -20,7 +18,7 @@ import type {ViewProps} from 'ViewPropTypes';
 
 const RCTMaskedView = requireNativeComponent('RCTMaskedView');
 
-type Props = {
+type Props = $ReadOnly<{
```
Comment (empyrical) [inline Libraries/Components/MaskedView/MaskedViewIOS.ios.js]: Should make it exact using `|`, like:

```js
type Props = $ReadOnly<{|
 //...
|}>;
```

File: Libraries/Components/MaskedView/MaskedViewIOS.ios.js [hunk 2]
```diff
@@ -20,7 +18,7 @@ import type {ViewProps} from 'ViewPropTypes';
...
   ...ViewProps,
 
   children: any,
```
Comment (empyrical) [inline Libraries/Components/MaskedView/MaskedViewIOS.ios.js]: `children` should be typed like this:

```js
children?: React.Node,
```

**Comment by nd-02110114** [comment]
Thank you for comments!  I fixed by following your comments.

**Comment by empyrical** [comment]
Thanks, LGTM! 👍

**Comment by TheSavior** [comment]
Thanks for contributing!

**Pull request closed without merging**